docs/custom_head.html

```diff
@@ -0,0 +1,17 @@
+<link rel="icon" href="https://realm.io/img/favicon.ico">
+    <link rel="apple-touch-icon-precomposed" sizes="57x57" href="https://realm.io/img/favicon-57x57.png" />
```

Comment:
indentation is like this on purpose to look nice in the generated HTML. I can remove the trailing `/`s though.